systems/bff/src/modules/packages/service.ts

```diff
@@ -0,0 +1,84 @@
+import { Service } from "typedi";
+import { catchAsyncIOMethod } from "../../common";
+import { IdResponse, ParsedCookie } from "../../common/types";
+import { API_METHOD_TYPE } from "../../constants";
+import { SERVER } from "../../constants/endpoints";
+import { checkError, HTTP404Error, Messages } from "../../errors";
+import { IPackageService } from "./interface";
+import PackageMapper from "./mapper";
+import {
+    AddPackageInputDto,
+    PackageDto,
+    PackagesResDto,
+    UpdatePackageInputDto,
+} from "./types";
+@Service()
+export class PackageService implements IPackageService {
+    getPackage = async (
+        packageId: string,
```

Comment:
The PackageId parameter is being passed but not utilized.
`pkgId` need to passed as a query params in url .

Reply:
Addressed